app/routes/circulars._archive._index/route.tsx

```diff
@@ -75,14 +81,54 @@ export default function () {
   if (searchString) searchString = `?${searchString}`
 
   const [inputQuery, setInputQuery] = useState(query)
-  const clean = inputQuery === query
+  const [inputDateGte, setInputDateGte] = useState(startDate)
+  const [inputDateLte, setInputDateLte] = useState(endDate)
+  const [showContent, setShowContent] = useState(false)
+  const [showDateRange, setShowDateRange] = useState(false)
+  const clean =
+    inputQuery === query &&
+    inputDateGte === startDate &&
+    inputDateLte === endDate
 
   const submit = useSubmit()
 
+  const setFuzzyTime = (startDate?: string) => {
+    if (startDate === 'custom') {
+      setShowDateRange(true)
+    } else {
+      setShowDateRange(false)
+      setInputDateGte(startDate)
+    }
+    setInputDateLte('')
```

Comment:
Factor this into two _different_ onClick handlers so that you don't need the `if (startDate === 'custom')` conditional logic.

Reply:
Ah, I actually think this might have been a holdover from when the radios were combined into one map and custom was one of the values it had to handle ([3c05f3c](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1482/commits/3c05f3cfd8ebc1718968e58f7c180ab0bb09bf67)). `setFuzzyTime()` doesn't actually handle the custom date radio button any more. I removed the conditional logic, but the call to `setShowDateRange(false)` is still present in [c9266c7](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1482/commits/c9266c7804380f15401426ec8519481bdd2c6142) to handle the case of selecting the custom date range but switching to a fuzzy date prior to submitting.